compiler/rustc_codegen_ssa/src/back/linker.rs

```diff
@@ -361,6 +343,38 @@ impl<'a> GccLinker<'a> {
 }
 
 impl<'a> Linker for GccLinker<'a> {
+    /// Passes an argument directly to the linker.
+    ///
+    /// When the linker is not ld-like such as when using a compiler as a linker, the argument is
+    /// prepended by `-Wl,`.
+    fn linker_arg(&mut self, arg: &OsStr, verbatim: bool) {
+        self.linker_args(&[arg], verbatim);
```

Comment:
Looks like this method just repeats the default implementation from `trait Linker`, so it shouldn't be necessary.